governance/polls/DAI Direct Deposit Module (D3M) Net Rates Spread - December 13, 2021.md

```diff
@@ -0,0 +1,52 @@
+---
+title: DAI Direct Deposit Module (D3M) Net Rates Spread - December 13, 2021
+summary: Signal your support or opposition to recommending a -0.5% net rates spread between AAVEv2 borrow rate and Maker ETH-A stability fee.
+discussion_link: https://forum.makerdao.com/t/signal-request-d3m-parameters-guidance-for-momc/11898
+vote_type: Plurality Voting
+categories:
+   - Rates
+   - Risk Variable
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2021-12-13T16:00:00
+end_date: 2021-12-16T16:00:00
+---
+# Poll: DAI Direct Deposit Module (D3M) Net Rates Spread - December 13, 2021
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the [Maker Open Market Committee](https://forum.makerdao.com/t/parameter-proposal-group-makerdao-open-market-committee/7355). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, December 13 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to recommending the following rates spread between AAVEv2 borrow markets and Maker ETH-A [stability fee](https://makerdao.world/en/learn/governance/param-stability-fee) for the [DAI Direct Deposit Module (D3M)](https://mips.makerdao.com/mips/details/MIP50): 
```

Comment:
MKR holders can vote according to template

Reply:
👍